### PR TITLE
Add model initializer config

### PR DESCRIPTION
**Pull request by yuzisun**
**What this PR does / why we need it**:
Currently model initializer image name and version is hard coded, moving it to configmap.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note
Add the model initializer image name/version to kfservice config map
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/244)

### CODE REVIEW

File: pkg/webhook/admission/deployment/model_initializer_injector_test.go [hunk 1]
```diff
@@ -594,3 +594,94 @@ func TestCredentialInjection(t *testing.T) {
...
+		},
+	}
+
+	for name, scenario := range scenarios {
+		injector := &ModelInitializerInjector{
+			credentialBuilder: credentials.NewCredentialBulder(c, &v1.ConfigMap{
+				Data: map[string]string{},
+			}),
+			config: &ModelInitializerConfig{
+				Image: "model-initializer:v0.1",
```
Comment (rakelkar) [inline pkg/webhook/admission/deployment/model_initializer_injector_test.go]: Can make this :vFromConfig or something so it's explicitly from config?

Comment (yuzisun) [reply pkg/webhook/admission/deployment/model_initializer_injector_test.go]: done!

**Pull request merged**